Plugin/Trajectory.cs

```diff
@@ -63,7 +63,9 @@ public struct Point
         public class Patch
         {
             public VesselState startingState { get; set; }
-            public double endTime { get; set; }
+            private double _endTime;

```

Comment:
Sorry, I don't quite understand. Why do we need two new variables? What's wrong with just using the old ones?